conda_recipes/aftereffects-saber/README.md

```diff
@@ -0,0 +1,49 @@
+# Saber plug-in conda build recipe for After Effects
+
+## About
+This recipe installs the Saber.aex plug-in file to the AE plugin directory located under `$PREFIX/opt/aftereffects/Plug-ins` in the Conda environment. To write conda recipes to add more plug-ins to your After Effects Deadline Cloud setup, use this recipe as a template to guide your development.
```

Comment:
nit: AE -> After Effects (AE). And then use AE instead of After Effects in the following paragraph. Or just using After Effects all places.